front/src/modules/rollingStock/components/RollingStockSelector/RollingStockSelector.tsx

```diff
@@ -1,35 +1,32 @@
 import React, { useRef } from 'react';
 import RollingStockModal from 'modules/rollingStock/components/RollingStockSelector/RollingStockModal';
+import RollingStock2Img from 'modules/rollingStock/components/RollingStock2Img';
 import icon from 'assets/pictures/components/train.svg';
 import {
   comfort2pictogram,
   RollingStockInfo,
 } from 'modules/rollingStock/components/RollingStockSelector/RollingStockHelpers';
 import { useModal } from 'common/BootstrapSNCF/ModalSNCF';
-import { RollingStockComfortType, RollingStock } from 'common/api/osrdEditoastApi';
+import { RollingStockComfortType, RollingStockWithLiveries } from 'common/api/osrdEditoastApi';
+import { useTranslation } from 'react-i18next';
 
 type RollingStockProps = {
-  rollingStockSelected?: RollingStock;
-  rollingStockComfort?: RollingStockComfortType;
-  image?: JSX.Element;
```

Comment:
If you remove this, this component won't have any image in storybook anymore andd we have an error in console